torch/onnx/utils.py

```diff
@@ -170,7 +170,8 @@ def _trace(func, args, operator_export_type, return_outs=False):
     if isinstance(args, torch.Tensor):
         args = (args, )
 
-    trace, torch_out = torch.jit.get_trace_graph(func, args, _force_outplace=True)
+    trace, torch_out = torch.jit.get_trace_graph(func, args,
```

Comment:
Changing this flag will change what operators are seen by the onnx transformer passes. I believe this will break a ton of export functionality that people are relying upon. I don't see any changes in this PR that would make this functionality work.